Zend/tests/offsets/runtime_compile_time_offset_access.phpt

```diff
@@ -0,0 +1,187 @@
+--TEST--
+Test binary operands exposing the same behavior at compile as at run time
+--INI--
+memory_limit=256M
+opcache.file_update_protection=1
+--SKIPIF--
+<?php
+//if (getenv("SKIP_SLOW_TESTS")) die('skip slow test');
+?>
+--FILE--
+<?php
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'test_offset_helpers.inc';
+
+function makeTestFile($container, $offset) {
```

Comment:
You could probably `file_get_contents('test_variable_offsets.inc')` and replace `$dimension`?

Reply:
Indeed